packages/router/router.ts

```diff
@@ -1547,7 +1585,7 @@ export function createRouter(init: RouterInit): Router {
     interruptActiveLoads();
     fetchLoadMatches.delete(key);
 
-    if (!match.route.action) {
+    if (!match.route.lazy && !match.route.action) {
```

Comment:
Need to slightly alter our "do I have an action or loader" logic since for lazy routes we can't know yet